nuxeo-mongodb-apb/meta/main.yml

```diff
@@ -14,7 +14,7 @@ galaxy_info:
   # - GPLv3
   # - Apache
   # - CC-BY
-  license: license (GPLv2, CC-BY, etc)
+  license: license (Apache)
```

Comment:
Shouldn't you just put Apache? What was in parenthesis were just examples I think.

Reply:
Of course. I will fix shortly.